libraries/botbuilder-dialogs-adaptive/src/actions/cancelAllDialogs.ts

```diff
@@ -8,10 +8,25 @@
 
 import { CancelAllDialogsBase } from './cancelAllDialogsBase';
 
+/**
+ * Command to cancel all of the current dialogs by emitting an event which must be caught to prevent cancelation from propagating.
```

Comment:
```suggestion
 * Command to cancel all of the current dialogs by emitting an event that must be caught to prevent cancellation from propagating.
```